js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
```

Comment:
어떤 DOM 요소를 가져오는지에 대한 정보가 변수명에 들어있으면 좋을 것 같은데요~
현재는 이메일에 관련된 에러인 것 같기는 한데 형태가 어떻게 되어있는지 파악하기 어려운 것 같아서 text, txt, message, msg와 같은 단어가 추가되면 더 괜찮을 것 같아요!

```suggestion
const $emptyEmailErrorMessage = QUERY_SELECTOR('#emptyEmailError');
const $wrongEmailErrorMessage = QUERY_SELECTOR('#wrongEmailError');
```